src/util/apiCalls.js

```diff
@@ -1,5 +1,5 @@
-const host = "https://blog-api.sida.dev/"
-//const host = "http://localhost:3001/"
+//const host = "https://blog-api.sida.dev/"
+const host = "http://localhost:3001/"
```

Comment:
Need to figure out how to use environment variables for this instead. This is too error prone. Not related to the PR but worth fixing now.